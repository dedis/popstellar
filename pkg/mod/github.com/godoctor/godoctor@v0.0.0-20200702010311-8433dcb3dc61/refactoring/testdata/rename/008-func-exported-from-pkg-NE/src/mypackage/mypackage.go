package mypackage

func MyFunction(n int) int { // <<<<< rename,3,6,3,6,xyz,pass
	if n == 0 {
		return 1
	} else {
		return n + MyFunction(n-1)
	}
}
