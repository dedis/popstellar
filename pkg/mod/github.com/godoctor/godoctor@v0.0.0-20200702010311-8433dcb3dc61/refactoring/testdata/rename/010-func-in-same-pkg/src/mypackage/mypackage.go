package mypackage

//Test for renaming a function with in same package
 
func MyFunction(n int) int {           // <<<<< rename,5,6,5,6,Xyz,pass  
	if n == 0 {
		return 1
	} else {
		return n + MyFunction(n-1)
	}
}
