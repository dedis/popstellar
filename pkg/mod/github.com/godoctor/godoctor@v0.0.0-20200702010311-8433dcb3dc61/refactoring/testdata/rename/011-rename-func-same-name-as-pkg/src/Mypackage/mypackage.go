package Mypackage

//Test for renaming a function with in same package
 
func Mypackage(n int) int {           // <<<<< rename,5,6,5,6,Xyz,pass  
	if n == 0 {
		return 1
	} else {
		return n + Mypackage(n-1)
	}
}
