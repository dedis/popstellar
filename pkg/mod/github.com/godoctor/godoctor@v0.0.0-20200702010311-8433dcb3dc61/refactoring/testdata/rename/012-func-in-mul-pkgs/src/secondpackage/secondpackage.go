package secondpackage 

import (
"mypackage"
)

func SecondFunction() int {             
	
    x := mypackage.MyFunction(10)
    
    x = x*100
  
   return x 
}

func IsEven(n int) bool {

	if n%2==0 {
  
		return true 
   	}

	return false

}
