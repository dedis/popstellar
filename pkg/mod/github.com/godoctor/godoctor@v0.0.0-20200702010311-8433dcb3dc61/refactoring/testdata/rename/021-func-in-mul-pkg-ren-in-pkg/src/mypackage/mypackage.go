package mypackage 
func MyFunction(n int) int {    // <<<<< rename,2,6,2,6,Xyz,pass         
	if n == 0 {
		return 1
	} else {
		return n + MyFunction(n-1)
	}
}
