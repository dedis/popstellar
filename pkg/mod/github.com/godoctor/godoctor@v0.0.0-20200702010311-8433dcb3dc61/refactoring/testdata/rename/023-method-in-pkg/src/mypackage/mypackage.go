package mypackage 


type Mystruct struct {

Myvar string

}

func (mystructvar *Mystruct)Mymethod() string {


return mystructvar.Myvar


}


