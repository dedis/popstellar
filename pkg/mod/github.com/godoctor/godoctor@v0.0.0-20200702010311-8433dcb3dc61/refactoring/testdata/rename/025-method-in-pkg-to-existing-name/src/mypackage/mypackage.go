package mypackage 


type Mystruct struct {

Myvar string

}

func (mystructvar *Mystruct)Mymethod() string {


return mystructvar.Myvar


}

func (mystructvar *Mystruct)Secondmethod() string {


return mystructvar.Myvar+"helloo"


}

