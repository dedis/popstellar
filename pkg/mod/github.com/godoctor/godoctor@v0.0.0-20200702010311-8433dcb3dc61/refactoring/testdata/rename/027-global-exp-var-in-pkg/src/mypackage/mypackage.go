package mypackage 


type Mystruct struct {

Myvar string

}

var Dumy string = "this is global exportable variable"
 
func (mystructvar *Mystruct)Mymethod() string {


return mystructvar.Myvar


}


