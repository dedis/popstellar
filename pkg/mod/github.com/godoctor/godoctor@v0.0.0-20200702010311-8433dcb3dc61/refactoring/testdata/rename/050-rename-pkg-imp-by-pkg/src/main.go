package main

import "fmt"
import "mypackage"
import "secondpackage"
//Test for renaming a package imported by other package and main file
func main() {                               
	fmt.Println(mypackage.MyFunction(5))
        fmt.Println(secondpackage.Simplesquare(9))     
}
