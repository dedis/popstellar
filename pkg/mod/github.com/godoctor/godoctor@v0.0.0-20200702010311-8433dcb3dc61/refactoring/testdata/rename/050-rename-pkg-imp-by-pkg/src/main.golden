package main

import "fmt"
import "mypackage"
import "Xyz"
//Test for renaming a package imported by other package and main file
func main() {                               
	fmt.Println(mypackage.MyFunction(5))
        fmt.Println(Xyz.Simplesquare(9))     
}
